### PR TITLE
Replace some other certificates by Jinja templates

### PR DESCRIPTION
**Pull request by inercia**
... like the `kubelet`, `dex` and `kube-proxy` ones (continues the work started in #280)

### CODE REVIEW

File: pillar/certificates.sls [hunk 1]
```diff
@@ -14,3 +14,37 @@ certificate_information:
   days_remaining:
     ca_certificate: 90
     certificate: 90
+
+ssl:
```
Comment (inercia) [inline pillar/certificates.sls]: Maybe we could remove this top `ssl` name by something more consistent: `certs_paths`, `certificates_paths`...

File: salt/dex/init.sls [hunk 2]
```diff
@@ -88,14 +39,14 @@ dex_secrets:
   cmd.run:
     - name: |
         until kubectl get secret dex-tls --namespace=kube-system ; do
-            kubectl create secret generic dex-tls --namespace=kube-system --from-file=/etc/pki/dex.crt --from-file=/etc/pki/dex.key
+            kubectl create secret generic dex-tls --namespace=kube-system --from-file={{ pillar['ssl']['dex_crt'] }} --from-file={{ pillar['ssl']['dex_key'] }}
```
Comment (inercia) [inline salt/dex/init.sls]: This will conflict with #285

**Comment by inercia** [comment]
> Do you think we could change the pillar structure, so for example, instead of having velum_key and velum_crt we could have:

Yeah, my idea was to refactor the pillar later on, after these PRs. This could allow us to just pass, for example, `certs="velum"`, as an argument and the Jinja template could take the `(key,crt)` pair from the pillar.

**Comment by ereslibre** [comment]
LGTM, will give it a try on monday.

**Pull request merged**